### PR TITLE
[+] Add Boar Vessel, 600-500 BC, Etruscan, ceramic (and forgery)

### PR DESCRIPTION
**Pull request by binarysudoku**
## About the PR 

Adds Boar Vessel, 600-500 BC, Etruscan, ceramic (and forgery)

## Why's this needed? 

What kind of self-respecting ~~museum~~ space station doesn't contain a forgery of Boar Vessel, 600-500 BC, Etruscan, ceramic

## Changelog 

(+) Added Boar Vessel, 600-500 BC, Etruscan, ceramic

```changelog
(u)Erinexx
(+) Added Boar Vessel, 600-500 BC, Etruscan, ceramic
```

### CODE REVIEW

**Review by Gannets — APPROVED** [review]
🐗

File: code/obj/item/misc_junk.dm [hunk 1]
```diff
@@ -642,3 +642,25 @@ TYPEINFO(/obj/item/reagent_containers/vape)
...
 	<br>
 	...spooky!"}
+
+/obj/item/boarvessel
+	name = "Boar Vessel, 600-500 BC, Etruscan, ceramic"
+	desc = "Oh my God! A REAL Boar Vessel, 600-500 BC, Etruscan, ceramic."
+	icon_state = "boarvessel"
+
+	attack_hand(mob/user)
+		if (user.a_intent == "help")
```
Comment (pali6) [inline code/obj/item/misc_junk.dm:652]: Note that this prevents picking it up on help intent. Perhaps only do this when in hand?

File: code/obj/item/misc_junk.dm [hunk 2]
```diff
@@ -642,3 +642,25 @@ TYPEINFO(/obj/item/reagent_containers/vape)
     The danger is unleashed only if you substantially disturb this place physically. This place is best shunned and left uninhabited.<br>
 	<br>
 	...spooky!"}
+
+/obj/item/boarvessel
+	name = "Boar Vessel, 600-500 BC, Etruscan, ceramic"
```
Comment (pali6) [inline code/obj/item/misc_junk.dm:647]:
```suggestion
	name = "\improper Boar Vessel, 600-500 BC, Etruscan, ceramic"
```

File: code/obj/item/misc_junk.dm [hunk 3]
```diff
@@ -642,3 +642,25 @@ TYPEINFO(/obj/item/reagent_containers/vape)
...
+	icon_state = "boarvessel"
+
+	attack_hand(mob/user)
+		if (user.a_intent == "help")
+			user.visible_message("<span class='alert'>[user] pets [src]!</span>")
+		else
+			..()
+
+/obj/item/boarvessel/forgery
+	name = "Boar Vessel, 600-500 BC, Etruscan, ceramic"
```
Comment (pali6) [inline code/obj/item/misc_junk.dm:658]:
```suggestion
	name = "\improper Boar Vessel, 600-500 BC, Etruscan, ceramic"
```

File: code/obj/item/misc_junk.dm [hunk 4]
```diff
@@ -642,3 +642,25 @@ TYPEINFO(/obj/item/reagent_containers/vape)
...
+	desc = "Whatever, it's probably not a REAL Boar Vessel, 600-500 BC, Etruscan, ceramic."
+
+	New()
+		. = ..()
+		src.AddComponent(/datum/component/radioactive, 1, FALSE, FALSE, 1)
```
Comment (pali6) [inline code/obj/item/misc_junk.dm:663]: Is this a good idea? 🐸

File: code/obj/item/misc_junk.dm (hunk 1 above)
Comment (binarysudoku) [reply code/obj/item/misc_junk.dm:652]: I'm not sure how to do this?

File: code/obj/item/misc_junk.dm (hunk 4 above)
Comment (binarysudoku) [reply code/obj/item/misc_junk.dm:663]: I have NO idea, I don't know how dangerous radioactive level 1 is, I just thought it was hilarious at the time

File: code/obj/item/misc_junk.dm (hunk 1 above)
Comment (DimWhat) [reply code/obj/item/misc_junk.dm:652]: Gnome chompsky has this behaviour if you are stumped!

Comment (binarysudoku) [reply code/obj/item/misc_junk.dm:652]: Ah! Thanks @DimWhat

File: code/obj/item/misc_junk.dm (hunk 4 above)
Comment (binarysudoku) [reply code/obj/item/misc_junk.dm:663]: Okay I've confirmed that there's essentially no rads being put into a player carrying this, so it literally just gives you spooky radiation messages without actually doing any harm whatsoever

File: code/obj/item/misc_junk.dm (hunk 1 above)
Comment (binarysudoku) [reply code/obj/item/misc_junk.dm]: Fixed issue in commit [216333e](https://github.com/goonstation/goonstation/pull/14545/commits/216333ea6f5364a08aeee14e087ac736408c6a39)

**Pull request merged**